dnf5-plugins/automatic_plugin/config/usr/share/dnf5/dnf5-plugins/automatic.conf

```diff
@@ -0,0 +1,106 @@
+# This configuration file is managed by the dnf5-plugin-automatic package.
+# Please do not edit it. To make changes in dnf5 automatic configuration
+# copy this file to /etc/dnf/dnf5-plugins/automatic.conf and make your
+# changes there.
```

Comment:
I did not change (for now) the logic how the config files are read, merely added a distribution config file to the package. So currently the /usr config file is not being read if the /etc one exists.
But I get your point. Let me think about it till tomorrow. Meanwhile I added a `blocked` label to the PR.

Reply:
Thanks @m-blaha FTR, no reason to block this PR, these are good changes. I was just wondering how it works. E.g. is it legit to create an /etc config with just a single key? If I set only `apply_updates = yes` in /etc, how will dnf-automatic treat e.g. the value of `upgrade_type`? As it works right now without *any* config file, I *guess* it has some builtin defaults anyway -- and hopefully they match the default file in /usr :grin:

Reply:
Yes, /usr config file just contains builtin defaults - thus it's perfectly OK to include just a single config option into /etc config file. But you are right that it's not very intuitive - user kind of expects that /usr is used as a baseline and then /etc overrides are applied on top of it.

Reply:
Thanks -- if the builtin defaults match the ones in /usr, that seems fine then. This would only mean that you can't change the one in /usr (ugh, never do that) *and* expect it to apply if you also have an /etc/ fiile. So this seems good enough?

Reply:
I did the change anyway (hopefully you don't mind the rebase). It seems more intuitive for the user that both config files are applied. It also enables us to adjust the default configuration in /usr if needed. Now everything should be in place and PR is ready for review.

Reply:
Very nice, also good fix for the random delay!

(FTR: rebases are how git works. I never understood some people's objections against it)